### ISSUE TITLE
[routing] Oneway for footway and steps.

### ISSUE THREAD

**Issue by vng** [organicmaps/organicmaps#5136]
Example: https://www.openstreetmap.org/#map=19/-54.84671/-68.48305

See also https://github.com/organicmaps/organicmaps/issues/1201#issuecomment-1519874163

**Comment by VladimirMorozov** [comment]
oneway should not be used for pedestrians and as such makes no sense on steps.
https://wiki.openstreetmap.org/wiki/Key:oneway#Pedestrians

It's even more important when a way allows both bicycles and pedestrians then oneway is for bicycles.

**Comment by biodranik** [comment]
@vng we're ignoring oneway=yes for walking mode, right?

**Comment by vng** [comment]
Yes, we ignore oneway for highways, but should not for designated pedestrian ways like steps, IMO.
See my initial example where peoples traffic is also important.

**Comment by 501Ghost** [comment]
There are over a thousand `oneway:foot=yes` tags and another 100+ `oneway:foot=-1` in OSM: https://taginfo.openstreetmap.org/keys/oneway:foot#values
Perhaps those tags can be considered.

**Comment by VladimirMorozov** [comment]
@vng My point was that your example is incorrectly tagged. Since organic maps is a popular app it would be nice if it didn't promote incorrect tagging. 

It's not really clear where you want to apply it. On steps only? On footways? On footways with no access set to anyone except foot?

**Comment by biodranik** [comment]
I agree that we should not support/promote incorrect tagging. https://wiki.openstreetmap.org/wiki/Key:oneway#Pedestrians clearly says that oneway=yes should be used only for cars.

@vng for your cases we need to support `oneway:foot=yes`, `foot:backward` and `foot:forward`. Our implementation may internally map them to our existing oneway implementation in the pedestrian context.

**Comment by Kamerijker** [comment]
Let me add the example of a separate entry and exit for pedestrians only, in a theme park. Examples are taken in Beersel Belgium Provinciaal domein Huizingen.
Entry and exit paths are coded oneway=yes, oneway:foot=no and foot:backward=no,(all redundant I agree). At least one would need to be supported in OM.
![Schermafbeelding 2024-02-15 om 10 53 58](https://github.com/organicmaps/organicmaps/assets/157891098/dc3a015a-791d-4b9d-b878-fbdff9348d16)
![OpenStreetMap](https://github.com/organicmaps/organicmaps/assets/157891098/7bcf4c69-220e-4f34-a96d-1bbf07ad9e77)
![Schermafbeelding 2024-02-15 om 10 55 17](https://github.com/organicmaps/organicmaps/assets/157891098/a224faf6-9796-4677-9e35-9915f0f8e44a)
The examples show OM getting in through the exit and getting out through the entry.
By preference foot:backward should be supported.
